.github/workflows/rust.yml

```diff
@@ -34,9 +34,9 @@ jobs:
       uses: cargo-bins/cargo-binstall@v1.12.3
     - name: Install loam
       run: cargo binstall loam-cli
-    - name: Build
-      run: cargo build --verbose
     - name: Build clients with Loam
       run: LOAM_ENV=testing loam build --build-clients
+    - name: Cargo build
+      run: cargo build --verbose
```

Comment:
Why both `loam build` and `cargo build`? Is there a non-contract crate in this workspace?

Reply:
No idea, Can I drop the `cargo build`? This PR simply moves it because the clients have to be built before `cargo build` happens

Reply:
Note; after I updated from main, it appears this is no longer a change related to this PR